spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestViews.java

```diff
@@ -1421,38 +1442,38 @@ public void showViews() throws NoSuchTableException {
     insertRows(6);
     String sql = String.format("SELECT * from %s", tableName);
     sql("CREATE VIEW v1 AS %s", sql);
-    sql("CREATE VIEW prefixV2 AS %s", sql);
-    sql("CREATE VIEW prefixV3 AS %s", sql);
+    sql("CREATE VIEW prefixv2 AS %s", sql);
```

Comment:
Hive Catalog returns table/view name always in lower case. To be safe changed the name to lowercase only.